contracts/eosio.system/include/eosio.system/guilds.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <eosio/asset.hpp>
+#include <eosio/multi_index.hpp>
+
+namespace eosiosystem {
+  static const time_point guild_bp_pay_start_time(eosio::seconds(1689379200));     // July 15st 2023 00:00:00
+  static constexpr int64_t  max_daily_guilds_fund = 321'000'00000000; // to_savings + to_per_block_pay per day
```

Comment:
Based on calculations from guilds we will set this to `800'000'00000000` initially